src/auth/TokenCache.ts

```diff
@@ -27,7 +27,7 @@ export class TokenCache {
     public async get(clientId: string, clientSecret: string): Promise<string | null> {
```

Comment:
not a big deal, but can the `async` be removed from the get / set methods now that you're using `read/writeFileSync`?

Reply:
yep, good catch 👀